examples/finished/logical.py

```diff
@@ -0,0 +1,64 @@
+from pyscipopt import Model
```

Comment:
can you write a comment saying what this should do, please?

Also, wouldn't make more sense to put this as a test? I guess examples are more for applications, something like solving a sudoku or whatnot.